src/app/datapoints-graph/datapoints-graph-view/datapoints-graph-widget-view.component.ts

```diff
@@ -48,7 +69,21 @@ export class DatapointsGraphWidgetViewComponent
       '"config" property should not be referenced in view component to avoid mutating data.'
     );
   }
+  @ViewChild(ChartsComponent) chartComponent!: ChartsComponent;
   displayConfig: DatapointsGraphWidgetConfig | undefined;
+  legendHelp = this.translate.instant(
+    gettext(`<ul class="m-l-0 p-l-8 m-t-8 m-b-0">
```

Comment:
I don't think that this whole string should be put in the gettext. It does not make sense to put <ul class="m-l-0 p-l-8 m-t-8 m-b-0"> etc

Reply:
why not? in our repo we do it. yeah, we could strip ul, but now it is whole tooltip and separating it (ul from li tags) would add confusion.
![image](https://github.com/user-attachments/assets/3b642f5d-1c3e-4233-babd-8f9c7d23fb15)